testData/inspections/suppression/fix/innerVariableDeclarationSuppressionFix2-after-highlighting.go

```diff
@@ -1,6 +1,6 @@
 package newmath
 
 func TestFunction() {
-	//noinspection GoUnusedVariable
```

Comment:
Maybe I've approached this the wrong way. I've split the GoUnusedVariable into two inspections but I think it can remain as a single inspection (since it doesn't matter if the variable is exported or not for a non-package level variable).

The problem would still happen for package level variables since we would still have two inspections (one for exported and one for unexported) in which case I don't know which one should be considered as a default for the existing `noinspection GoUnusedGlobalVariable`.